pkg/controller/registry/reconciler/configmap.go

```diff
@@ -90,9 +91,56 @@ func (s *configMapCatalogSourceDecorator) Service() *v1.Service {
 }
 
 func (s *configMapCatalogSourceDecorator) Pod(image string) *v1.Pod {
-	pod := Pod(s.CatalogSource, "configmap-registry-server", image, s.Labels(), 5, 2)
-	pod.Spec.ServiceAccountName = s.GetName() + ConfigMapServerPostfix
-	pod.Spec.Containers[0].Command = []string{"configmap-server", "-c", s.Spec.ConfigMap, "-n", s.GetNamespace()}
+	pod := &v1.Pod{
```

Comment:
should we update the configmap based catalogs if these store raw manifests and don't require `opm` to unpack bundles IIRC (since they don't support bundles)?